ModelImporter/import_scene.py

```diff
@@ -373,24 +369,16 @@ def render_scene(self):
 
     def _add_armature_to_scene(self):
         """ Each joint will be added as an armature. """
+        scene = SceneOp(bpy.context)
```

Comment:
Might be better to do `self.scene_op = SceneOp(bpy.context)` (eg.) maybe in `__init__` so we don't need to call it for every single node that is imported...